modules/operators/migrations/20_0_0-beta_1-tap-response/index.ts

```diff
@@ -0,0 +1,94 @@
+import { Rule, Tree, SchematicContext } from '@angular-devkit/schematics';
+import {
+  visitTSSourceFiles,
+  createReplaceChange,
+  commitChanges,
+  Change,
+} from '@ngrx/component/schematics-core';
```

Comment:
This import can be a relative path to the `schematics-core` folder within the operators package.